voting.tex

```diff
@@ -1,80 +1,133 @@
+\chapter{Voting Procedure}
+\label{cha:voting-procedure}
+
 \section{Voting Methods}
-The voting methods for all chapter related activity shall be restricted to the systems listed below.
-All methods require quorum
-Unanimous Vote
-All active and voting brothers must vote to pass the given proposal in order for it to be approved.
-Super-Majority Vote
-At least three fourths of the active, and voting brothers must vote to pass the given proposal in order for it to be approved.
-Simple Majority Vote
-More than one half of the active, and voting brothers must vote to pass the given proposal in order for it to be approved.
-Instant Runoff Vote
-Each active and voting brother ranks a pool of candidates in order of preference.
-The votes are tallied, and the candidate with the fewest first choice votes is eliminated.
-If two or more candidates are tied for the fewest first choice votes, the eliminated candidate is determined by second choice and so on, until the tie can be broken.
-If the candidates remain tied for last, then all such candidates are eliminated.
-The votes are re-tallied, with votes for eliminated candidates being counted towards the next-highest choice on that ballot.
-This process is repeated until one candidate is the highest remaining choice on more than half of the remaining ballots.
-Plurality Vote
-The proposal with the most supporting votes from active, and voting brothers will be approved.
+\label{sec:voting-methods}
+
+The voting methods for all chapter related activity shall be restricted to the
+systems listed below.
+All methods require quorum.
+
+\begin{itemize}
+	\item Unanimous Vote
+		\begin{itemize}
+			\item All active and voting brothers must vote to pass the given
+				proposal in order for it to be approved.
+		\end{itemize}
+	\item Super-Majority Vote
+		\begin{itemize}
+			\item At least three fourths of the active, and voting brothers must
+				vote to pass the given proposal in order for it to be approved.
+		\end{itemize}
+	\item Simple Majority Vote
+		\begin{itemize}
+			\item More than one half of the active, and voting brothers must
+				vote to pass the given proposal in order for it to be approved.
+		\end{itemize} \item Instant Runoff Vote \begin{itemize} \item Each
+				active and voting brother ranks a pool of candidates in order of
+				preference.
+				The votes are tallied, and the candidate with the fewest first
+				choice votes is eliminated.
+				If two or more candidates are tied for the fewest first choice
+				votes, the eliminated candidate is determined by second choice
+				and so on, until the tie can be broken.
+				If the candidates remain tied for last, then all such candidates
+				are eliminated.
+				The votes are re-tallied, with votes for eliminated candidates
+				being counted towards the next-highest choice on that ballot.
+				This process is repeated until one candidate is the highest
+				remaining choice on more than half of the remaining ballots.
```

Comment:
Since this is a list of steps, should we make this an enumerate?

Reply:
It's a bullet list in the original document.

Reply:
Then shouldn’t this be a bucket list?

Reply:
Bullet